pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/convert_to_raw_index/ConvertToRawIndexTaskExecutor.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.minion.executor;
+package org.apache.pinot.plugin.minion.tasks.convert_to_raw_index;
```

Comment:
Would it be good to use all lower cases instead of snake cases for the package name here?

Reply:
do you mean `org.apache.pinot.plugin.minion.tasks.converttorawindex`?
I was trying to follow: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
I see. Thanks for providing the reference here. If we want to follow this convention, it'd be good to refactor other package paths in the repo as well. E.g.:
`org.apache.pinot.broker.requesthandler` for `BaseBrokerRequestHandler` class.